webpack/scenes/ContentViews/Details/Versions/ContentViewVersions.js

```diff
@@ -96,7 +96,7 @@ const ContentViewVersions = ({ cvId }) => {
       cells={columnHeaders}
       variant={TableVariant.compact}
       autocompleteEndpoint={`/content_view_versions/auto_complete_search?content_view_id=${cvId}`}
-      fetchItems={params => getContentViewVersions(cvId, params)}
+      fetchItems={useCallback(params => getContentViewVersions(cvId, params), [cvId])}
```

Comment:
Do we also need this getContentViewVersions in the Dependency array here?

Reply:
It's defined outside of the component, so it's not needed in the dependency array according to the eslint rule.